staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -328,6 +333,47 @@ func (g *gcePersistentDiskCSITranslator) GetCSIPluginName() string {
 	return GCEPDDriverName
 }
 
+func (g *gcePersistentDiskCSITranslator) RepairVolumeHandle(volumeHandle, nodeID string) (string, error) {
+	var err error
+	tok := strings.Split(volumeHandle, "/")
+	if len(tok) != volIDTotalElements {
+		return "", fmt.Errorf("volume handle has wrong number of elements; got %v, wanted %v", len(tok), volIDTotalElements)
+	}
+	if tok[volIDProjectValue] != UnspecifiedValue {
+		return volumeHandle, nil
+	}
+	
+	nodeTok := strings.Split(nodeID, "/")
+	if len(nodeTok) != volIDTotalElements {
```

Comment:
is this right? node token has the same number of elements as volume id?

Reply:
yes